sdk-lib/src/main/java/cash/z/ecc/android/sdk/Synchronizer.kt

```diff
@@ -277,12 +277,15 @@ interface Synchronizer {
     suspend fun validateAddress(address: String): AddressType
 
     /**
-     * Download all UTXOs for the given address and store any new ones in the database.
+     * Download all UTXOs for the given account addresses and store any new ones in the database.
      *
-     * @return the number of utxos that were downloaded and addded to the UTXO table.
+     * @param account The Account, for which all addresses blocks will be downloaded.
+     * @param since The BlockHeight, from which blocks will be downloaded.
+     *
+     * @return the number of utxos that were downloaded and added to the UTXO table.
      */
     suspend fun refreshUtxos(
-        tAddr: String,
+        account: Account = Account.DEFAULT,
```

Comment:
I'm skeptical of default arguments; they're almost always a bad idea and lead users down unpleasant paths.

Reply:
I generally agree with no default arguments (except in our test fixtures and occasionally in Compose UI method signatures)

In this particular case, I think we should keep the account default argument until the SDK actually supports accounts.

Reply:
The problem here is that you have multiple methods that all provide a default for their argument, which is dangerous: all call sites have to choose to either supply an argument, or to use the default. I'd much prefer that the callers be modified to be called with `Account.DEFAULT` than to leave this in place - it will prove a land mine when adding account support to the SDK.

Reply:
I agree and recommend this as a [followup issue](https://github.com/zcash/zcash-android-wallet-sdk/issues/939), so that we can get this PR merged which will fix an important bug in the SDK.

Reply:
Yes, I agree. Thank you for pointing this out.